x-pack/plugins/monitoring/server/kibana_monitoring/lib/source_kibana.js

```diff
@@ -19,14 +19,15 @@ const snapshotRegex = /-snapshot/i;
  * @param {String} host Kibana host
  * @return {Object} The object containing a "kibana" field and source instance details.
  */
-export function sourceKibana(kbnServer, config, host) {
+export function sourceKibana(server, kbnServer) {
```

Comment:
I realize `sourceKibana` name reflects the `source_node` naming from Elasticsearch, and maybe it makes more sense there, but I think here it's a somewhat unhelpful name in this context. I think something like `getKibanaInfoForStats()` is a bit more helpful to people curious what this function does.